app/src/main/java/com/example/harvestfresh/LoginActivity.java

```diff
@@ -21,15 +22,22 @@ public class LoginActivity extends AppCompatActivity {
     private EditText etUsername;
     private EditText etPassword;
     private Button btnLogin;
+    private Button btnSignup;
+    final FragmentManager fragmentManager = getSupportFragmentManager();
 
     @Override
     public void onCreate(@Nullable Bundle savedInstanceState) {
         super.onCreate(savedInstanceState);
         setContentView(R.layout.activity_login);
 
+//        if(ParseUser.getCurrentUser() != null){
```

Comment:
Same comment as earlier about commented code

Reply:
Done! I had removed but it only after this!